tests/test-common.js

```diff
@@ -276,7 +276,7 @@ if(options.earl && options.earl.filename) {
   options.earl.report = new EarlReport({id: options.earl.id});
 }
 
-return new Promise((resolve, reject) => {
+return new Promise(resolve => {
```

Comment:
Personally, I like to keep the parenthesis around as it just reads more clearly and when you add a second param you have to put them back anyhow. But that's probably a primarily personal preference. 😃 

The config is [arrow-parens](https://eslint.org/docs/rules/arrow-parens#always) fwiw:
```
/*eslint arrow-parens: ["error", "always"]*/
```

Otherwise, 👍!

Reply:
Yeah, there's a permathread in the community on whether or not to always have the parenthesis. So it's a stalemate on this one.

The main counter to "it's easier to refactor" is "basing your style on something *you may do* isn't as good an idea as basing it on what you *do do*". For example, there's a lot of simple functional programming stuff like mapping arrays `foo.map(x => y)` where needlessly adding parenthesis actually makes things harder to read and more annoying to type.

Anyway, we'd sided with that faction ... for now. :smile: